tests/test_api.py

```diff
@@ -1506,6 +1489,69 @@ def verify_build_json(passed_build_json):
         build_response = osbs._create_scratch_build(build_request)
         assert build_response.json == {'spam': 'maps'}
 
+    @pytest.mark.parametrize(('nodeselector_str, nodeselector_dict'), [
+        (None, {}),
+        ('foo=  bar', {'foo': 'bar'}),
+        ('foo=bar ,  baz=foo', {'foo': 'bar', 'baz': 'foo'}),
+    ])
+    def test_scratch_build_nodeselector(self, nodeselector_str, nodeselector_dict):
+        if nodeselector_str:
+            with NamedTemporaryFile(mode='wt') as fp:
+                fp.write(dedent("""\
+                    [general]
+                    build_json_dir = {inputs}
+                    [default]
+                    low_priority_node_selector = {nodeselector}
+                    """.format(inputs=INPUTS_PATH, nodeselector=nodeselector_str)))
+                fp.flush()
+                config = Configuration(fp.name)
+        else:
+            config = Configuration()
+        osbs = OSBS(config, config)
```

Comment:
Added `noqa` to this line - these are test fixtures and actually are used.

I've fixed other pep8 errors in this file